inc/vcf/normalizer.hpp

```diff
@@ -27,64 +27,94 @@ namespace ebi
      */
     struct RecordCore
     {
-        size_t line;
+        std::string line;
+        size_t line_num;
         std::string chromosome;
         size_t position;
         std::string reference_allele;
         std::string alternate_allele;
         RecordType alternate_allele_type;
-        
-        RecordCore(size_t line, const std::string &chromosome, size_t position, 
-                   const std::string &reference_allele, const std::string &alternate_alleles, RecordType alternate_allele_types) 
-            : line(line), chromosome(chromosome), position(position), 
-              reference_allele(reference_allele), alternate_allele(alternate_alleles),
-              alternate_allele_type(alternate_allele_types)
+
+        RecordCore(size_t line_num,
+                   const std::string & chromosome,
+                   size_t position,
+                   const std::string & reference_allele,
+                   const std::string & alternate_alleles,
+                   RecordType alternate_allele_types
+                   ) : line_num(line_num),
+                       chromosome(chromosome),
+                       position(position),
+                       reference_allele(reference_allele),
+                       alternate_allele(alternate_alleles),
+                       alternate_allele_type(alternate_allele_types)
+        {
+            line = "";
+        }
```

Comment:
In C++11 you can make one constructor call another one, which makes sure that for a given set of parameters they will behave the same way.